Cereal/CerealDecoder.swift

```diff
@@ -42,14 +42,14 @@ public struct CerealDecoder {
     private let reverseRange: StrideThrough<Int>
     private let indexMap: [String: Int]?
 
-    private func itemForKey(key: String) -> CoderTreeValue? {
+    fileprivate func itemForKey(key: String) -> CoderTreeValue? {
```

Comment:
This should be `itemForKey(_ key: String) -> CoderTreeValue?`. We shouldn't need to duplicate the word `key`.